packages/web/src/scss/components/UNSTABLE_Slider/index.html

```diff
@@ -0,0 +1,223 @@
+{{#> web/layout/plain }}
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="UNSTABLE_Slider">
+      <label for="slider-default" class="UNSTABLE_Slider__label">
+        Slider
+      </label>
+      <input
+        class="UNSTABLE_Slider__input"
+        id="slider-default"
+        max="100"
+        min="1"
```

Comment:
Can we set `min` to `0` or delete it to keep the default value of the range input and not collide with the react demo implementation (in all examples)?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#specifying_the_minimum_and_maximum